pkg/volume/flexvolume/unmounter.go

```diff
@@ -50,7 +50,8 @@ func (f *flexVolumeUnmounter) TearDownAt(dir string) error {
 	}
```

Comment:
I think the PathExists implementation is wrong.  It's returning "false" (path doesn't exist) for any error except for corrupted mount errors.  https://github.com/kubernetes/kubernetes/blob/master/pkg/util/mount/mount_helper_common.go#L101

This will probably leave orphaned directories.

Reply:
@msau42 good catch, if it's a corrupted mnt point, `PathExists` should always returns `true, err`, I have made the changes, and it now works well for unmounting corrupted smb mnt point. PTAL.

Reply:
Out of pure curiosity - now that we have changed behaviour of `PathExists`, should we audit and double check all other code is calling it correctly and expecting correct behaviour?

Reply:
@gnufied @msau42 finally I have time to fix this issue (because of customer pushing ^_^).
I changed this PR to fix the stuck issue only in flexvol driver code. When hit corrupted mnt path, it `mount.PathExists(dir)`  returns `<false, error>`  and with this PR, it only do logging and then invoke `NewDriverCall(unmountCmd)`.

PTAL, thanks.